### ISSUE TITLE
version.txt and version-ltr.txt contain outdated info about the currently available QGIS versions

### ISSUE THREAD

**Issue by agiudiceandrea** [qgis/QGIS-Website#372]
### URL

https://version.qgis.org/version.txt

### Summary

https://version.qgis.org/version.txt and https://version.qgis.org/version-ltr.txt, retrieved by QGIS in order to know if there is a new QGIS version, contain outdated info about the currently available QGIS version:

```
#QGIS Version 33800|Visit https://download.qgis.org to get your copy of version 3.38.0|<html><body>
<ul>
<li>The current released version of QGIS is 3.38.0.
<li>We are working on 3.39-Master. It will go into feature freeze on 2024-09-13 12:00:00 UTC and be released as 3.40 on 2024-10-25 12:00:00 UTC.
<li>Details on changes can be found on github at <a href="https://github.com/qgis/QGIS/commits/master">https://github.com/qgis/QGIS/commits/master</a>.
</body>
</html>
```

```
#QGIS Version 33408|Visit https://download.qgis.org to get your copy of version 3.34.8|<html><body>
<ul>
<li>The current available LTR version of QGIS is 3.34.8.
<li>The latest available version of QGIS is 3.38.0.
<li>We are working on 3.39-Master. It will go into feature freeze on 2024-09-13 12:00:00 UTC and be released as 3.40 on 2024-10-25 12:00:00 UTC.
<li>Details on changes can be found on github at <a href="https://github.com/qgis/QGIS/commits/master">https://github.com/qgis/QGIS/commits/master</a>.
</body>
</html>
```

**Comment by agiudiceandrea** [comment]
See also https://github.com/qgis/QGIS/issues/58255.

**Comment by agiudiceandrea** [comment]
@jef-n, version.txt and version-ltr.txt still contain outdated info. Maybe the regeneration of such files should be somehow triggered on the server.

**Comment by jef-n** [comment]
Fixed with qgis/QGIS-Sysadmin-Documentation#61
